draft-ietf-scitt-architecture.md

```diff
@@ -376,6 +376,23 @@ The SCITT architecture consists of a very loose federation of Transparency Servi
 In order to accommodate as many Transparency Service implementations as possible, this document only specifies the format of Signed Statements (which must be used by all Issuers) and a very thin wrapper format for Receipts, which specifies the Transparency Service identity and the agility parameters for the Signed Inclusion Proofs.
 Most of the details of the Receipt's contents are specified in the COSE Signed Merkle Tree Proof document {{-COMETRE}}.
 
+SCITT consists of the following roles: Issuers, Transparency Services, Verifiers, and Auditors.
+The process works as follows:
+
+1. Issuers make Statements about Artifacts
+1. Issuers sign the Statements, producing Signed Statements
+1. Issuers submit the Signed Statements to Transparency Services
+1. Transparency Services take the submitted Signed Statements and insert them into an Append-only Log, producing Transparent Statements.
+This process can be thought of as Transparency Services counter-signing Issuers' Signed Statements
+1. Verifiers consume the Transparent Statements, and by verifying them, are convinced of the authenticity and integrity of Statements about an Artifact
+1. Auditors verify the correctness and consistency of the Transparent Statements
+
+Note that Issuers commit to a Statement by signing it and by submitting it to a Transparency Service.
+They are not necessarily the first ones nor the only ones who publish a Statement.
+
+An entity can take on multiple roles.
```

Comment:
For the various pieces of feedback above, you may want to split out multiple roles from above. 
Note: we are having a conversation about terminology associated with RATS and other IETF Specs. See #120 and the latest [IETF SCITT Meeting notes](https://datatracker.ietf.org/wg/scitt/meetings/)

Reply:
Since that discussion is ongoing, I would leave it as-is for now. And update it later in a separate PR :)